config/commands.js

```diff
@@ -1932,7 +1933,8 @@ var commands = exports.commands = {
 			}
 			return this.sendReplyBox("Random number " + num + "x(1 - " + faces + "): " + rolls.join(", ") + "<br />Total: " + total);
 		}
-		if (target && isNaN(target) || target.length > 21) return this.sendReply("The max roll must be a number under 21 digits.");
+		if (isNaN(target) || target >= 1e22) return this.sendReply("The max roll must be a number under 21 digits.");
+		if (target <= 0) return this.sendReply("Number of faces must be greater than 0.");
 		var maxRoll = (target) ? target : 6;
```

Comment:
This command currently requires target to run, but this line makes me suspect it wasn't always that way. Should I just remove this line and switch the following cases of ```maxRoll``` to ```target``` ?